src/wp-includes/pluggable.php

```diff
@@ -2840,6 +2837,18 @@ function get_avatar( $id_or_email, $size = 96, $default_value = '', $alt = '', $
 			$args['width'] = $args['size'];
 		}
 
+		/**
+		 * Update args with loading optimized attributes.
+		 *
+		 * @since 6.3
+		 */
```

Comment:
```suggestion
		// Update args with loading optimized attributes.
```
This doesn't document an action or filter, so it doesn't need a docblock or `@since` annotation.